packages/kbn-code-owners/src/file_code_owner.ts

```diff
@@ -65,12 +65,10 @@ export function getPathsWithOwnersReversed(): PathWithOwners[] {
 export function getCodeOwnersForFile(
   filePath: string,
   reversedCodeowners?: PathWithOwners[]
-): string | undefined {
+): { path: string | undefined; teams: string | undefined } {
```

Comment:
I think it would be better to stay with `undefined` in case no match was found. Makes it easier to consume the result. So I'd suggest to make this something like
```suggestion
): { path: string; teams: string } | undefined {
```